test/units/module_utils/basic/test_exit_json.py

```diff
@@ -7,20 +7,21 @@
 import json
 import sys
 import datetime
+import typing as t
 
 import pytest
 
 from ansible.module_utils.common import warnings
 
-EMPTY_INVOCATION = {u'module_args': {}}
+EMPTY_INVOCATION: dict[str, dict[str, t.Any]] = {u'module_args': {}}
```

Comment:
```suggestion
EMPTY_INVOCATION: dict[str, dict[t.Never, t.Never]] = {u'module_args': {}}
```

Reply:
For this, and other suggestions on refining the type annotations further, feel free to work on them in another PR. The goal of this PR was to enable mypy on unit tests. I fixed some of the easier issues in the tests with some basic annotations, and added ignores for the rest.